### DISCUSSION TITLE
Resume Bundle X ATS SCORING

### DISCUSSION THREAD

**Discussion by Aryan9901** [recodehive/Stackoverflow-Analysis#365] [Q&A] [answered]
Hello sir, My name is Aryan Gupta and i am pursing btech from computer science engineering. I want to ask you that when I am going to apply for a company sometime my resume got rejected every time 

I want to also ask that beside all this tech I know what else I can learn or what type of projects I should make 

[Aryan_Gupta.pdf](https://github.com/user-attachments/files/17005713/Aryan_Gupta.pdf)

[Aryan Gupta Ats friendly.pdf](https://github.com/user-attachments/files/17005715/Aryan.Gupta.Ats.friendly.pdf)

Sir here are my two resume I used interchangeably

My linkedin profile is 
https://www.linkedin.com/in/agaryan

My portfolio is 
https://www.aryang.vercel.app

![Aryan Gupta Ats friendly.pdf](https://github.com/user-attachments/files/17005765/Aryan.Gupta.Ats.friendly.pdf)

![Aryan_Gupta.pdf](https://github.com/user-attachments/files/17005766/Aryan_Gupta.pdf)

**Comment by farhansadikgalib** [comment, marked as answer]
Your portfolio and resume are really great.

**Comment by Ojha8421** [comment]
Hey Aryan,

Here’s a quick breakdown:

Why Rejections Might Happen:
ATS Systems: Your resume might not be ATS-friendly. Use simple formatting and add job-specific keywords.
Tailor Your Resume: Customize it for each job and highlight relevant skills.
Show Impact: Focus on what you achieved, not just what you built. Add numbers to show results.

What to Learn:
Cloud/DevOps: Learn AWS, Docker, Kubernetes—companies love cloud skills.
System Design: Understand how large-scale systems work for better interviews.
Open Source: Contribute to open-source projects to gain experience.

Project Ideas:
Full-Stack Apps: Build practical apps like a to-do list with React/Django.
Cloud Projects: Deploy something to AWS or GCP.
Automation: Automate tasks with scripts or pipelines.

Resume Tips:
Use ATS-friendly format, add job-specific keywords, and focus on project impact (use numbers if possible).

**Comment by yazansedih** [comment]
The resume seems good.

**Comment by TatyOko28** [comment]
Hi Aryan,
Thanks for sharing your resumes and profile! It's great that you're actively applying and looking to improve your chances. Here are some key points to consider:

1. Resume Optimization for ATS
Ensure your resume is ATS-friendly by using a clean, text-based format (avoid fancy designs, tables, or graphics).
Use relevant keywords from the job descriptions you’re applying to (skills, tools, and technologies mentioned in the JD).
Prioritize quantifiable achievements (e.g., "Developed an application that reduced processing time by 30%" instead of just "Built an app").
Keep your skills section concise and relevant. Avoid listing outdated or unrelated technologies.
2. What Else Can You Learn?
Since you're pursuing BTech in CSE, apart from technical skills, companies also value:

Problem-solving & DSA: Focus on LeetCode, CodeChef, GeeksforGeeks (many companies use DSA-based coding rounds).
System Design Basics: If applying for backend roles, understanding scalability, databases, API design helps.
Soft Skills & Communication: Many rejections happen due to poor interview communication. Practicing mock interviews will help.
3. What Projects to Build?
Depending on your interests, here are some project ideas that can make your resume stand out:

Web Development: A full-stack project using React + Node.js + MongoDB/PostgreSQL.
AI/ML: A predictive model using Python, TensorFlow, or OpenCV.
Cloud Computing: Deploy a project on AWS, GCP, or Azure to showcase DevOps & cloud skills.
Blockchain: A simple smart contract project using Solidity & Ethereum.
4. Profile & Portfolio Review
✅ Your LinkedIn looks professional, but consider adding:

More detailed descriptions in the experience & project sections.
Engage with industry leaders, post your learnings & projects.
✅ Your portfolio website looks great! Make sure it includes:

A concise "About Me" section focusing on your strengths.
A blog or project breakdown where you discuss challenges & solutions.
Next Steps:

Try using a customized resume for each job by adjusting keywords to match the job description.
Keep practicing DSA for coding rounds.
Improve your LinkedIn engagement (comment, post, network with recruiters).
Focus on one or two impactful projects and refine them instead of multiple incomplete ones.
Hope this helps! Let me know if you need a detailed resume review or interview tips. 🚀

Best of luck with your job applications!

Regards,
Taty Oko
